apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs

```diff
@@ -299,6 +304,42 @@ function writeNodeRow(row, nodeStatus, buttonType, addlClass) {
     %>
     </div>
 
+    <% if (areLobbyHintsEnabled() && controller.isUserAlone()) { %>
+    <div class="lobby-hint">
+      <i class="fa fa-info-circle" aria-hidden="true"></i>
+      <p>
+        You're the only one here!  If you were expecting to see other people
+        here, you might have accidentally joined the wrong simulator instance.
+        Follow these steps to make sure you're in the right place:
+      </p>
+      <ul>
+        <li><strong>Are you signed in?</strong>
+        To use the simulator with your class, you must be
+        signed in to a Code Studio account that is in the same class section as
+        the other students.  Have your teacher confirm that your account is
+        in the class section.</li>
+        <li><strong>Are you on the right level?</strong>
+        You'll only be able to see other students in your class if you are on the
+        same simulator level.  Make sure everyone is on <%= window.location %></li>
```

Comment:
nit: it looks a bit strange to print the url as plain text. Could we do something like wrap it in a `<pre>`?